app/controllers/concerns/activity_tracking.rb

```diff
@@ -30,4 +30,8 @@ def track_recovered(trackable, user: current_user, project: current_project)
   def track_updated(trackable, user: current_user, project: current_project)
     track_activity(trackable, :update, user, project)
   end
+
+  def track_updated_state(trackable, user: current_user, project: current_project)
```

Comment:
Don't love the name here, and looking at the presenter further down, what'd be the point of splitting the methods, and the action if we're not going to differentiate the view?

It seems to me that what we care about it's not the updating on an attribute, but the change in the QA state. Naming convention around QA state, and displaying a related message would be more useful

Reply:
Updated the name to `track_state_change`. 
We are differentiating the view [here](https://github.com/dradis/dradis-ce/pull/1134/files#diff-99b696dd220f1c79d35442c8fce53b3177ba41ed9377401e1b725a715afa2cc5R3)